interview-questions.md

```diff
@@ -5,54 +5,55 @@ First, without external resources. Challenge yourself to answer from memory.
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely on your own, there is always something more to learn. Write your researched answer in your OWN WORDS.  
 
-1. What are props in React?
+1. What are props in React? 
 
-  Your answer:
+  Your answer: Props are communication arroyos that allow information to flow downstream (one direction) from parent to child components. 
```

Comment:
I think you mean "communication arrays" here like a radio tower antenna. Props are like the documents given to a private when he is assigned a task by a superior. the documents can be accessed and read and used accordingly. Any authority to act with said information comes down from above. 

Props are key:value pairs that are assigned to child components upon creation by the parent component.